tensorbay/cli/utility.py

```diff
@@ -189,3 +188,86 @@ def error(message: str) -> NoReturn:
     """
     click.secho(f"ERROR: {message}", err=True, fg="red")
     sys.exit(1)
+
+
+def update_config() -> None:
+    """Update the config file to the new format."""
+    old_config_parser = read_config()
+    old_sections = old_config_parser.sections()
+    if not old_sections or old_sections == ["config"]:
+        return
+
+    if (
+        old_sections in (["config", "profiles"], ["profiles"])
```

Comment:
what happens if `old_sections` is `["profiles", "config"]`?

Reply:
It will be considered not updated.